src/forgot-password/ForgotPasswordPage.jsx

```diff
@@ -28,16 +27,15 @@ import ForgotPasswordAlert from './ForgotPasswordAlert';
 import messages from './messages';
 
 const ForgotPasswordPage = (props) => {
+  const platformName = getConfig().SITE_NAME;
+  const emailRegex = new RegExp(VALID_EMAIL_REGEX, 'i');
   const {
     intl, status, submitState, emailValidationError,
   } = props;
 
-  const platformName = getConfig().SITE_NAME;
-  const supportUrl = getConfig().LOGIN_ISSUE_SUPPORT_LINK;
-  const regex = new RegExp(VALID_EMAIL_REGEX, 'i');
-
-  const [bannerEmail, setBannerEmail] = useState('');
   const [email, setEmail] = useState(props.email);
+  const [bannerEmail, setBannerEmail] = useState('');
+  const [formErrors, setFormErrors] = useState('');
```

Comment:
Can we use `validationError` state instead of introducing `formErrors` because we can achieve our goal for `ForgetPasswordAlert` with it too?

Reply:
No. If we use validationError it will continuously show the error alert but we only want to show it once the user has submitted the form.
Also if the validation error changes, the header error will also change which should not happen.

Reply:
Gotcha. Thankyou!